dts/bindings/dma/intel,sedi_dma.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel SEDI DMA controller.
+
+compatible: "intel,sedi_dma"
+
+include: dma-controller.yaml
+
+properties:
```

Comment:
reg and interrupts are used in SEDI DMA driver, so need to add them as required. or they're optional in DTS.

 11   reg:                                                                                              
 12     required: true                                                                                  
 13                                                                                                     
 14   interrupts:                                                                                       
 15     required: true